source/quickstart.txt

```diff
@@ -252,4 +254,32 @@ Synchronization Notes
 - To estimate the size of ``oplog`` needed for initial synchronization,
   see :ref:`c2c-oplog-sizing`.
 
+.. _c2c-quickstart-many-with-one:
+
+Sync Multiple Source Clusters Simultaneously with a Destination Cluster
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: /includes/many-with-one-cluster.rst
+
+For limitations, see :ref:`c2c-limitations_multiple_sync`.
+
+The following examples connect source clusters running on port 27000 and
```

Comment:
**Suggestion**

I think the port numbers here should be monospaced, i.e `27000`, `27001` and `35000`.

Reply:
Done.